HashSet.py

```diff
@@ -0,0 +1,24 @@
+class MyHashSet:
+
+    def __init__(self):
+        self.mp=defaultdict(int) # Defaultdict is dictionary, combination of key value pair and keys would be unique which is the requirement for hashset
```

Comment:
for designing any data structure , you cannot use the same inbuilt data structure.